src/style/addbook.css

```diff
@@ -0,0 +1,40 @@
+.addform {
```

Comment:
- [ ] Great job implementing the styles, however kindly consider making your app like the Zeplin design, and don't forget all the elements in the design should be added (`comment`, `edit`, and the little separator). also, the nav bar links should be aligned to the left. this will make your design more complete and professional

|Your design|Figma design|
|-|-|
|![Screenshot_20220617_001416](https://user-images.githubusercontent.com/25227022/174194047-c571b779-14d0-4298-8fe0-0b544b42b63b.png)|![Screenshot_20220617_002253](https://user-images.githubusercontent.com/25227022/174194061-a292ef4b-14c1-4081-b726-0af6e4af2b48.png)|